src/command/control/stop.c

```diff
@@ -78,7 +78,8 @@ cmdStop(void)
                     // The file is locked so that means there is a running process - read the process id and send it a term signal
                     char contents[64];
                     ssize_t actualBytes = read(fd, contents, sizeof(contents));
-                    String *processId = actualBytes > 0 ? strTrim(strNewN(contents, (size_t)actualBytes)) : NULL;
+                    String *processId = actualBytes > 0 ?
+                        strTrim(strLstGet(strLstNewSplitZ(strNewN(contents, (size_t)actualBytes), LF_Z), 0)) : NULL;
```

Comment:
Should be, but the file could be invalid or in the process of being written. In that case we'll just ignore it.

Perhaps we should add a retry here, but I don't think it is needed because the process taking the lock will check for a stop file next.